Source/WebCore/dom/DocumentOrShadowRoot.idl

```diff
@@ -44,4 +44,7 @@ interface mixin DocumentOrShadowRoot {
     Element? elementFromPoint(double x, double y);
     sequence<Element> elementsFromPoint(double x, double y);
     // CaretPosition? caretPositionFromPoint(double x, double y); // FIXME: Implement this.
+
+    // https://fullscreen.spec.whatwg.org/#api
+    [LegacyLenientSetter, Conditional=FULLSCREEN_API, EnabledBySetting=UnprefixedFullscreenAPIEnabled] readonly attribute Element? fullscreenElement;
```

Comment:
This should be in its own file as a `partial interface mixin`

Reply:
Ok.